sdk/purestorageblock/arm-purestorageblock/test/public/purestorageblock_operations_test.spec.ts

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ *
+ * Changes may cause incorrect behavior and will be lost if the code is regenerated.
+ */
+
+import { env, Recorder, isPlaybackMode } from "@azure-tools/test-recorder";
+import { createTestCredential } from "@azure-tools/test-credential";
+import { assert, beforeEach, afterEach, it, describe } from "vitest";
+import { createRecorder } from "./utils/recordedClient.js";
+import { BlockClient } from "../../src/blockClient.js";
+
+export const testPollingOptions = {
+  updateIntervalInMs: isPlaybackMode() ? 0 : undefined,
+};
+
+describe("purestorageblock test", () => {
+  let recorder: Recorder;
+  let subscriptionId: string;
+  let client: BlockClient;
+
+  beforeEach(async (context) => {
+    process.env.SystemRoot = process.env.SystemRoot || "C:\\Windows";
+    recorder = await createRecorder(context);
+    subscriptionId = env.SUBSCRIPTION_ID || "";
+    // This is an example of how the environment variables are used
+    const credential = createTestCredential();
+    client = new BlockClient(credential, subscriptionId, recorder.configureClientOptions({}));
+  });
+
+  afterEach(async function () {
+    await recorder.stop();
+  });
+
+  it.skip("operations list test", async function () {
```

Comment:
skip this case as api version is not available:
```
"ResponseBody": {
        "error": {
          "code": "InvalidResourceType",
          "message": "The resource type 'operations' could not be found in the namespace 'PureStorage.Block' for api version '2024-11-01-preview'. The supported api-versions are '2023-06-01-preview,2024-06-01-preview,2024-08-01-preview,2024-10-01-preview'."
        }
      }
```
fyi @deepakmauryams

Reply:
why this is a green field service if this is not the first preview api version?